tap_mailchimp/discover.py

```diff
@@ -1,22 +1,52 @@
+import json
+import os
+from singer import metadata
 from singer.catalog import Catalog, CatalogEntry, Schema
+from tap_mailchimp.streams import STREAMS
 
-from tap_mailchimp.schema import get_schemas, PKS
+def get_abs_path(path):
```

Comment:
These kind of functions should be part of schema.py
Ref: https://github.com/singer-io/tap-linkedin-ads/blob/crest-master/tap_linkedin_ads/schema.py

Reply:
Updated the discovery code as per suggested.